.github/test_sdk/main.cxx

```diff
@@ -8,7 +8,7 @@
 
 int main()
 {
-  f3d::engine eng(f3d::engine::FLAGS_NONE);
+  f3d::engine eng(f3d::window::WindowType::NO_RENDER);
```

Comment:
`window::WindowType` is weird. What about renaming the enum just `type` or `Type`?